### PR TITLE
feat: add Expeditions button, modal teaser

### LINKED ISSUE #1132: Expeditions modal + button
Task doesn't include adding any logic to subfeatures.

### PR DESCRIPTION
**Pull request by denciu**
# Summary

Closes #1132 

- Buttons in cards will be probably connected with logic per feature so treat these only as examples.
- Changes balances to a new compact design by @zamli.
- Add notation to account balances:
  image
  image

### CODE REVIEW

**Comment by wixzi** [comment]
Screenshot 2022-06-27 at 17 53 00

May be push everything little down in mobile view or ask zett for a mobile design.

**Comment by denciu** [comment]
> Screenshot 2022-06-27 at 17 53 00
> 
> May be push everything little down in mobile view or ask zett for a mobile design.

Yup, Zett is already thinking about it.

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -308,6 +312,11 @@ function Header() {
         </HeaderSubRow>
 
         <Flex maxHeight={'22px'} justifyContent={'end'}>
+          {account && (
+            <HeaderButton onClick={toggleExpeditionsPopup} style={{ marginRight: '7px' }}>
+              &#128640;&nbsp;Expeditions
```
Comment (jorgedclopes) [inline src/components/Header/index.tsx:317]: what the hell are these chars? Seems worthy adding them to a variable on the side.
ok... I googled and then understood them but I don't think anyone will understand them just by looking at the code.

**Comment by adamazad** [comment]
> 

@zamli could you track this please?

**Pull request merged**